pyproject.toml

```
@@ -25,6 +25,7 @@ classifiers = [] # https://pypi.org/classifiers/
 dependencies = [
     "grpcio ~= 1.53.0",
     "grpcio_tools ~= 1.53.0",
+    "prometheus_client >= v0.18.0"
```

That allows us to use v1.0 whenever that be published, and even if that breaks something. I'd rather pin this to a major release:
```suggestion
    "prometheus_client ~= v0.18.0"
```

---

ACK